userdocs/diagnostics/multiple-inheritance.md

```diff
@@ -0,0 +1,51 @@
+# Multiple Inheritance
+
+In some programming languages, a class can inherit the interface of multiple base classes. Known as multiple inheritance, this feature can add significant complexity to the language and is unsupported in Swift. Instead, Swift allows composition of interfaces using protocols.
+
+Consider the following example:
+
+```swift
+protocol Utensil { 
+    var name: String { get }
+} 
+
+/* ------------------------------ Serving Utensil ------------------------------ */
+protocol ServingUtensil: Utensil {
+    func serve()
+} 
+
+extension ServingUtensil {
+    func serve() { /* ... */ }
+}
+
+/* ------------------------------ Fork ------------------------------ */
+protocol Fork: Utensil {
+    func spear()
+}
+
+extension Fork {
+    func spear() { /* ... */ }
+}
+
```

Comment:
```suggestion
```

I don't think we need to demonstrate implementing a default implementation more than once. Let's cut out anything that doesn't help the user understand the concept of multiple inheritance and how to use protocol composition instead.